ParabolInc/parabol
# feat(standups): Activities from other team members get real-time updates

Fixes: #6395 

## Important info
indicator 'XYZ is adding a response' will be a separate task
This PR depends on https://github.com/ParabolInc/parabol/pull/6388. 

## Tests

1. Updates
- create a standup meeting, join from 2 browsers
- update the response, click away (currently mutation is fired in `onBlur`), make sure the other user sees the update
- update the response as another user, click away (currently mutation is fired in `onBlur`), and make sure the first one sees the update

2. New team member
- copy the team invite link
- create & join the standup meeting
- open incognito, accept the invite 
- make sure the first user sees the new card for a new team member being added

https://user-images.githubusercontent.com/1017620/167173342-bf914132-f910-4e28-970c-7d8bc6f7cbad.mp4
